tests/test_dynamic_metadata.py

```diff
@@ -236,24 +235,23 @@ def test_pep517_wheel(virtualenv):
     assert wheel == dist / out
 
     virtualenv.install(wheel)
-    if sys.version_info >= (3, 8):
```

Comment:
FYI, I may update all these tests to not use `zipfile.Path`, it's actually a little better not to use it, I've done one or two. That would cut down on the total cleanup a bit.

Reply:
I did this, and rebased this PR.